app/renderer/src/assets/locales/en/translation.json

```diff
@@ -281,5 +281,6 @@
   "noResultsFound": "No results found",
   "invalidCapType": "Invalid capability type: {{type}}",
   "whitespaceDetected": "Text Starts and/or Ends With Whitespace",
-  "pcloudyCredentialsRequired": "pCloudy username and api key are required!"
+  "pcloudyCredentialsRequired": "pCloudy username and api key are required!",
+  "duplicateCapabilityNameError": "Capability with the same name already present"
```

Comment:
The error text seems a bit confusing, one might think he used the same capability name such as platformName. Rather than the capability set settings

Reply:
@Dor-bl Got it. How about rephrasing is as `Capability Set with the same name already present`? Open for suggestions.